content/2016.12.how-to-develop-a-webapp-with-Kinto.md

```diff
@@ -125,19 +125,21 @@ In order to get messages, we will have to write a bit of code to get the room's
 
 ```js
 // https://discute.re/#4815162342  is mapped to the collection 4815162342
-this.collectionName = window.location.hash.slice(1);
+collectionName = window.location.hash.slice(1);
 ```
 
-Then we need to do a GET request to get the list of all room's messages.
+Then we need to do a `GET` request to get the list of all room's messages.
 A simple `GET https://kinto.discute.re/v1/buckets/chat/collections/4815162342/records` would do the job (with fetch or XMLHttpRequest), but we will rather be using the [kinto-https-js](https://github.com/kinto/kinto-http.js) library that offers a clean client API, take a look at this code:
 
-    const kinto_url = 'https://kinto.discute.re/v1';
-    // options stores the connection identifiers
-    const client = new KintoClient(kinto_url, options);
-    client.bucket('chat')
-        .collection('4815162342')
-        .listRecords()
-        .then(({data}) => displayMessage(data));
+```js
+const kinto_url = 'https://kinto.discute.re/v1';
```

Comment:
It seems we've already started using snakeCase with `collectionName` above, so this should probably be `kintoUrl` for consistency